mnist/README.md

```diff
@@ -34,6 +34,13 @@
  - Repos->Outputs->Model: select mnist and enter mountpath as /model
  - Submit
 
+## Hyperparameter Tuning
+1.Create a Run same as explained above, except that now a tuning file also needs to be uploaded in the configuration tab.
+  - For katib hyperparameter tuning upload the https://github.com/oneconvergence/dkube-examples/blob/tensorflow/mnist/katib-mnist-tuning.json under upload  configuration. 
+  - For simple hyperparameter tuning upload the https://github.com/oneconvergence/dkube-examples/blob/tensorflow/mnist/tuning.yaml under upload configuration.
+  - Submit the run. 
```

Comment:
What is the difference between both the tuning ?

tuning.yaml also creates a Katib run

Reply:
yes @deepio-oc my bad,  i have updated.

Reply:
Could have remove the json file and  use tuning.yaml ? Update the instructions accordingly.

YAML is better format for specifying configuration.

Reply:
Okay done @deepio-oc. we can merge the pr now.